package.json

```diff
@@ -95,6 +96,14 @@
   "workspaces": [
     "e2e/*",
     "packages/*",
-    "samples/*"
-  ]
+    "samples/*",
+    "lib/*"
+  ],
+  "dependencies": {
+    "core-js": "^3.19.2",
+    "nx": "^13.1.4",
+    "regenerator-runtime": "0.13.7",
+    "rxjs": "^7.4.0",
+    "tslib": "^2.0.0"
+  }
```

Comment:
Did we have to add these back? I thought we moved them to the respective workspace/project.

Reply:
yup must have been a bad rebase from old state, nice catch